cores/esp8266/core_esp8266_main.cpp

```diff
@@ -60,6 +60,13 @@ static os_event_t s_loop_queue[LOOP_QUEUE_SIZE];
 /* Used to implement optimistic_yield */
 static uint32_t s_micros_at_task_start;
 
+/* For ets_intr_lock_nest / ets_intr_unlock_nest
+ * Max nesting seen by SDK so far is 2.
+ */
+#define ETS_INTR_LOCK_NEST_MAX 7
+uint16_t ets_intr_lock_stack[ETS_INTR_LOCK_NEST_MAX];
```

Comment:
Probably want these new vars as `static` here, just to match the rest of the locals.

Reply:
yep, good point. I'll do it once we get some positive reports back.
I've been accessing them from some of my other test code, but once that's no longer needed they can go static.